dags/telemetry_aggregates_fennec_backfill.py

```diff
@@ -0,0 +1,24 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2016, 12, 15),
+    'email': ['frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 3,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('telemetry_aggregates', default_args=default_args, schedule_interval='@daily')
+
+t0 = EMRSparkOperator(task_id = "telemetry_aggregate_fennec_backfill",
```

Comment:
I'm gonna be that PEP-8 jerk -- no spaces around the equal sign for keyword arguments ;)